federation/BUILD

```diff
@@ -26,6 +26,7 @@ filegroup(
         "//federation/cmd/genfeddocs:all-srcs",
         "//federation/cmd/kubefed:all-srcs",
         "//federation/develop:all-srcs",
+        "//federation/pkg/crud:all-srcs",
```

Comment:
Naming nit: crudmachinery? crudtestmachinery? I worry that crud isn't a descriptive enough name, and could be mistaken for something that deals with CRUD operations in federation production code rather than as a test harness.